.ci/deps.cabal.sh

```diff
@@ -7,6 +7,8 @@ cabal --version
 
 cabal update
 
+cabal install happy
```

Comment:
I believe this file isnt being used atm, and thus this change isnt being tested and should be removed.  Perhaps note the change on https://github.com/coala/coala-bears/issues/1384 , as this was likely a step towards fixing it

Reply:
nope; `cabal` is used as a selector in tox.ini